src/interfaces/UploadResponse.ts

```diff
@@ -159,4 +159,8 @@ export interface UploadResponse {
    * Field object which will contain the status of each extension at the time of completion of the update/upload request.
    */ 
   extensionStatus?: { [key: string]: string }
+  /*
+   * Metadata for the response containing headers and status
+   */
+  $ResponseMetadata?: object
```

Comment:
Don't add it to uploadResponse, use the IKResponse Type Modifier

Refer to node.js SDK index.ts file